### PR TITLE
Branding object detail

### PR DESCRIPTION
**Pull request by Doppen**

### CODE REVIEW

File: apps/researcher/src/app/[locale]/(objects)/objects/[id]/metadata.tsx [hunk 1]
```diff
@@ -105,14 +105,24 @@ export async function MetadataEntry({
   const t = useTranslations('ObjectDetails');
   const format = await getFormatter();
 
+  if (isCurrentPublisher && !children) {
```
Comment (barbarah) [inline apps/researcher/src/app/[locale]/(objects)/objects/[id]/metadata.tsx:108]: This is moved to line 36. Can you remove this block and add the classnames to the code in line 36?

**Comment by barbarah** [comment]
Looking at the preview server: the background is still white, is this on purpose? 

image

**Comment by Doppen** [comment]
Should have the right colors now
> Looking at the preview server, the background is still white. Is this on purpose?
> 
> image

File: apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx [hunk 2]
```diff
@@ -112,171 +112,178 @@ export default async function Details({params}: Props) {
...
-      </div>
 
-      <div className="px-4 sm:px-10 my-4 flex flex-col gap-4 w-full bg max-w-[1800px] mx-auto">
-        <h1
-          className="text-2xl md:text-4xl md:items-center"
-          data-testid="page-title"
-        >
-          {object.name}
-        </h1>
+        <div className="px-4 sm:px-10 my-4 flex flex-col gap-4 w-full bg max-w-[1800px] mx-auto">
```
Comment (barbarah) [inline apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx:130]: The title here has the classname `max-w-[1800px]`. But this class is missing at other divs. So, on big screens, only this div has a max width, which looks weird.

image

File: apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx [hunk 3]
```diff
@@ -112,171 +112,178 @@ export default async function Details({params}: Props) {
...
+                {object.inscriptions?.map(inscription => (
+                  <div key={inscription}>{inscription}</div>
+                ))}
+              </Metadata>
+            </div>
+          </main>
+          <aside className="w-full md:w-1/3 self-stretch order-1 md:order-2  md:mx-0 md:bg-consortiumBlue-900 p-1">
+            {galleryImages.length > 0 && (
+              <div className="flex flex-row md:flex-col gap-1 sticky top-4">
+                <Gallery images={galleryImages} />
```
Comment (barbarah) [inline apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx:280]: The license of images has been added recently. The grey tile colour is not readable on the blue background.

File: apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx [hunk 4]
```diff
@@ -112,171 +112,178 @@ export default async function Details({params}: Props) {
...
+              </div>
+            )}
+          </aside>
+        </div>
+        <Provenance objectId={id} />
```
Comment (barbarah) [inline apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx:285]: `Provenance` (newly added) is not designed yet.

**Review by barbarah — APPROVED** [review]
I'm going to approve this for now, I think it good enough for a first version. However there are a few things I see that can be fixed in a next pull request.

**Pull request merged**